certora/README.md

```diff
@@ -1,56 +1,60 @@
 # Running the certora verification tool
 
-These instructions detail the process for running CVT on the OpenZeppelin (Wizard/Governor) contracts.
+These instructions detail the process for running Certora Verification Tool on the OpenZeppelin contracts.
 
-Documentation for CVT and the specification language are available
-[here](https://certora.atlassian.net/wiki/spaces/CPD/overview)
+Documentation for CVT and the specification language are available [here](https://certora.atlassian.net/wiki/spaces/CPD/overview)
+
+## Prerequisites
+
+Follow the [Certora installation guide](https://docs.certora.com/en/latest/docs/user-guide/getting-started/install.html) in order to get the Certora Prover Package and the `solc` executable folder in your path
+
+> **Note**
+> An API Key is required for local testing. Although the prover will run on a Github Actions' CI environment on each Pull Request.
```

Comment:
We are not running it in every pull request to avoid too high resource usage.

```suggestion
> An API Key is required for local testing. Although the prover will run on a Github Actions' CI environment on select Pull Requests.
```

Reply:
Wouldn't be `selected Pull Requests`?

Reply:
Both work grammatically but `selected` seems to have the right meaning.